R/calculate_stats.R

```diff
@@ -0,0 +1,263 @@
+calculate_stats <- function(seasons = nflreadr::most_recent_season(),
+                            summary_level = c("season", "week"),
+                            stat_type = c("player", "team")){
+
+  # testing
+  # seasons = 2023
+  # summary_level = "week"
+  # stat_type = "player"
+
```

Comment:
Do we want a default summary level (season?) and stat type (player?)

And error checking for valid summary level / stat type provided?

Reply:
Both argument defaults and inputs checks are done in lines 10+11